### ISSUE TITLE
Action to generate .di header file from a .c file using ImportC

### ISSUE THREAD

**Issue by SingingBush** [intellij-dlanguage/intellij-dlanguage#906]
As discussed in this forum thread https://forum.dlang.org/post/ugvc3o$5t3$1@digitalmars.com, dmd can generate di files (essentially D header files) from C code:

```
dmd -c test.c -Hf=test.di
```

It would be great (and reasonably straight forward) to have an action available when right-clicking on a _*.c_ file that prompts the user for a destination output (defaulting to same file name but with .di extension in same directory) then attempts to use dmd based on the syntax above.

 - Without dmd configured the option should be visible in the context menu but not enabled.
 - This is dmd only!
 - There will need to be a good level of error handling and the terminal output should probably be made visible to the user.

**Comment by rikkimax** [comment]
In saying that, this probably won't do what you want it to. You'd be missing any include paths ext. and for Windows we still haven't solved the whole needing the VS vars to be loaded in.

Basically, fully self-contained files might work, but that's about it.

**Comment by SingingBush** [comment]
understood. it'll be limited but I think it's worth getting some initial support in for this feature and work through pitfalls over time. Managing user expectations will be important so the UI will need to indicate limitations somehow.

**Comment by rikkimax** [comment]
Yes, it's a happy accident that it works even in the simplest of cases. The main thing here is that us going into it must keep it described as limited. Otherwise, people are gonna be unhappy!
